### ISSUE TITLE
Cant select species when Building Recogniser

### ISSUE THREAD

**Issue by tomevison** [smarsland/AviaNZ#91]
Hi There, I am having some issues training a Wavelet Recogniser.

Currently have version AviaNZ 3.2 installed on Ubuntu 20.04.4 LTS.

When I try to train a Wavelet Recogniser, I have my selected files, but the "Select Species" dropdown box does not contain any selectable values.

Kindest regards,

![image](https://user-images.githubusercontent.com/17394368/176196039-d3563fe6-b72e-44fd-a530-d58f165bf8ab.png)

**Comment by smarsland** [comment]
Sorry, it's taken me too long to look at a lot of these (especially sorry to you, David!).
You do need labelled data to train the classifier, so the workflow we expect is that you make manual labels of your species in some set of files (say 10-20 calls) and then try the training. The wavelet classifier itself will (usually) do a reasonable job of finding examples of your species along with lots of false positives. These can then be the basis of some training for the neural network and other advanced methods. This requires lots and lots of training data annotations. For the Fiordland tokoeka one we had several hundred hours of recordings. We are trying to do various other things to make classifiers that work with less data, but so is the rest of the world!
Regards,
Stephen
